testprojects/src/python/docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python:3.8
+ENTRYPOINT ["/main.pex"]
+COPY testprojects.src.python.hello.main/main.pex /
```

Comment:
@benjyw do you think this is OK syntax to base dep inference on?

Example works, at least ;)
```
    $ ./pants package testprojects/src/python/docker/Dockerfile
    13:57:59.08 [INFO] Completed: Building testprojects.src.python.hello.main/main.pex with 1 requirement: ansicolors==1.1.8
    13:58:01.24 [INFO] Completed: Building docker image test-example:1.2.5
    13:58:01.24 [INFO] Built docker image: test-example:1.2.5
    To try out the image interactively: docker run -it --rm test-example:1.2.5 [entrypoint args...]
    To push your image: docker push test-example:1.2.5

    $ docker run -it --rm test-example:1.2.5 Andy Drew
    Hello, Drew!
    Hello, Andy!
```

Reply:
Assuming that is the relative path of the pex in the digest, then that seems correct?